tests/sentry/integrations/opsgenie/test_integration.py

```diff
@@ -119,3 +124,287 @@ def test_update_config_invalid(self):
         assert installation.get_config_data() == {
             "team_table": [{"id": team_id, "team": "cool-team", "integration_key": "1234"}]
         }
+
+
+ALERT_LEGACY_INTEGRATIONS = {
+    "id": "sentry.rules.actions.notify_event.NotifyEventAction",
+    "name": "Send a notification (for all legacy integrations)",
+}
```

Comment:
Rather than copy paste, you can probably import this variable from the file to keep them in sync.

Reply:
This is a constant that I created. Not sure if there's a variable.